docs/HOL_Provisioning-Deploying-Environments-Using-Chef/HOL_Provisioning-Deploying-Environments-Using-Chef.md

```diff
@@ -6,7 +6,7 @@ In this hands-on lab you will explore some of the new features and capabilities
 
 - An SSH client such as PuTTY
 
-- PartsUnlimitedMRP deployed to an Azure Linux Virtual Machine (see [https://github.com/Microsoft/PartsUnlimitedMRP/blob/master/docs/Build-MRP-App-Linux.md](https://github.com/Microsoft/PartsUnlimitedMRP/blob/master/docs/Build-MRP-App-Linux.md))
```

Comment:
you need to use the raw.githubusercontent... link in order for all of these to work